src/Raven.Server/ServerWide/Commands/Subscriptions/PutSubscriptionCommand.cs

```diff
@@ -51,16 +52,27 @@ public override unsafe void Execute(TransactionOperationContext context, Table i
                         var ptr = tvr.Read(2, out int size);
                         var doc = new BlittableJsonReaderObject(ptr, size, context);
 
-                        var subscriptionState = JsonDeserializationClient.SubscriptionState(doc);
+                        var existingSubscriptionState = JsonDeserializationClient.SubscriptionState(doc);
 
-                        if (SubscriptionId != subscriptionState.SubscriptionId)
-                            throw new InvalidOperationException("A subscription could not be modified because the name '" + subscriptionItemName + "' is already in use in a subscription with different Id.");
-                        
+                        if (SubscriptionId != existingSubscriptionState.SubscriptionId)
+                            throw new InvalidOperationException("A subscription could not be modified because the name '" + subscriptionItemName +
+                                                                "' is already in use in a subscription with different Id.");
+
+                        if (InitialChangeVector == "DoNotChange")
```

Comment:
Yeah, but we've decided to use this command for subscription definition updates in the first place. Do you think that we need to have a special command for this?